docs/docs/features-reference.md

```diff
@@ -25,7 +26,7 @@ Here will come a filter panel to search by categories, what's strategic, and mor
 
 **🏁 CLI flags:** --logger=true|false
 
-## 1.1.2 Prevent infinite logger serialization loop
```

Comment:
I love the better hierarchy, wouldn't you keep the number so people can refer to features ("Issue - In feature 1.2 you forgot to...")?

Reply:
I agree it can help.
I will add numbering for the `###` headlines as well.